lute/std/libs/test.luau

```diff
@@ -57,8 +57,8 @@ end
 -- Test Reporting
 type FailedTest = {
 	test: string, -- name of the test case that failed
-	assertion: string, -- name of the assertion that failed (e.g., "assert.eq")
-	error: string, -- the error message from the assertion
+	assertion: string?, -- name of the assertion that failed (e.g., "assert.eq")
+	error: string?, -- the error message from the assertion
```

Comment:
nit: It might still be useful to leave this non-optional and broaden it from `errors` for assertions to `errors` for `FailedTests`. I think letting the user know something like: `"Runtime error in your test function"` is still valuable.